chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/QuestionableActivityEmailJob.java

```diff
@@ -135,10 +132,7 @@ public void execute(final JobExecutionContext jobContext) throws JobExecutionExc
             recipients.add(to);
 
             EmailBuilder emailBuilder = new EmailBuilder(env);
-            emailBuilder.recipients(recipients)
-                    .subject(subject)
-                    .htmlMessage(htmlMessage)
-                    .fileAttachments(files)
+            emailBuilder.recipients(recipients).subject(subject).htmlMessage(htmlMessage).fileAttachments(files)
```

Comment:
Not sure why this file was modified, and I'm okay with most of these formatting changes, but, at least for the builders, I know I find those to be much more readable when each building component is on a new line